styles/styles.css

```
@@ -0,0 +1,243 @@
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:400 ;
+    src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:700 ;
+    src: url("../fonts/rubik/rubik-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family:"Raleway" ;
+    font-style: normal;
+    font-weight:800 ;
+    src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
+    font-display: swap;
+}
+
+body {
+    font-family: "Rubik";
+    font-style: normal;
+    font-size:18px ;
```

```suggestion
    font-style: normal;
    font-size: 18px ;
```
у тебя где-то есть пробелы, а где-то нету. сделай везде красиво, чтобы было с пробелом